### PR TITLE
Make a backup before updating

### PR DESCRIPTION
**Pull request by WhiteCoreWebUI**
To Provide the ability to return to a previously working System if something goes wrong.

### CODE REVIEW

File: update.sh [hunk 1]
```diff
@@ -1,7 +1,10 @@
 # Check for any updates of Linux
 sudo apt-get update
-# Update WhiteCore-Sim
+# go to /
```
Comment (fly-man-) [inline update.sh]: I don't see why this is included ....

**Comment by emperorstarfinder** [comment]
That really is actually not needed.  When your updating it just checks to see if there are updates from the latest master.  If your looking to make a backup of Whitecore prior to updating which I think is what was intended here then you would want to make a compressed archive of whitecore before checking for any updates.  #go to / does not package or back anything up that is essentially just saying cd to the root directory.

**Comment by WhiteCoreWebUI** [comment]
it goes to / to make a whole backup of the Github folder it could also go to /Github and make a backup of WhiteCore-Dev the cd / was already in the update script i simply make a backup before it goes into /Github

File: update.sh [hunk 2]
```diff
@@ -1,7 +1,10 @@
...
 cd /
+# Run backup of current Working WC System
+tar czf backup.$(date +%Y%m%d-%H%M%S).tar.gz Github
```
Comment (emperorstarfinder) [inline update.sh]: The concern here I believe is that this backs up the entire github directory.  It really should just back up the WhiteCore-Dev directory as the user might end up using the Github directory for other repositories to.

**Comment by WhiteCoreWebUI** [comment]
Updated the Script to backup only the WhiteCore-Dev path

**Pull request merged**